Source/Web/app/information.js

```diff
@@ -425,7 +482,7 @@ function dispOrch(id) {
           "" +
```

Comment:
Assigned Workers don't seem to show up. I was testing with Apple Farm - Block H Orchard.